functions/java-jre.bash

```diff
@@ -104,9 +111,31 @@ java_zulu_8_tar(){
   jdkBin=$(find "${jdkInstallLocation}"/*/bin ... -print -quit)
   jdkLib=$(find "${jdkInstallLocation}"/*/lib ... -print -quit)
   cond_redirect update-alternatives --remove-all java
+  cond_redirect update-alternatives --remove-all jjs
+  cond_redirect update-alternatives --remove-all keytool
+  cond_redirect update-alternatives --remove-all orbd
+  cond_redirect update-alternatives --remove-all pack200
+  cond_redirect update-alternatives --remove-all policytool
+  cond_redirect update-alternatives --remove-all rmid
+  cond_redirect update-alternatives --remove-all rmiregistry
+  cond_redirect update-alternatives --remove-all servertool
+  cond_redirect update-alternatives --remove-all tnameserv
+  cond_redirect update-alternatives --remove-all unpack200
+  cond_redirect update-alternatives --remove-all jexec
   ## TODO: remove sometime in late 2020
   cond_redirect update-alternatives --remove-all javac
   cond_redirect update-alternatives --install /usr/bin/java java "$jdkBin"/java 1083000
+  cond_redirect update-alternatives --install /usr/bin/jjs jjs "$jdkBin"/jjs 1083000
```

Comment:
That magic number 108300 is totally arbitrary. I guess it was 100 000 once and was more or less accidentially increased by 100 or 1000 every time someone touched java code. Latest myself IIRC.
Maybe a good opportunity to reset it ?

Reply:
Sure, I just left it because there was no reason to change it.